airflow/decorators/__init__.pyi

```diff
@@ -726,6 +726,94 @@ class TaskDecoratorCollection:
         self, python_callable: Callable[FParams, FReturn] | None = None
     ) -> Task[FParams, FReturn]: ...
     @overload
+    def pyspark_submit(
+        self,
+        *,
+        multiple_outputs: bool | None = None,
+        use_dill: bool = False,  # Added by _PysparkSubmitDecoratedOperator.
+        conf: dict[str, Any] | None = None,
+        conn_id: str = "spark_default",
+        files: str | None = None,
+        py_files: str | None = None,
+        archives: str | None = None,
+        driver_class_path: str | None = None,
+        jars: str | None = None,
+        java_class: str | None = None,
+        packages: str | None = None,
+        exclude_packages: str | None = None,
+        repositories: str | None = None,
+        total_executor_cores: int | None = None,
+        executor_cores: int | None = None,
+        executor_memory: str | None = None,
+        driver_memory: str | None = None,
+        keytab: str | None = None,
+        principal: str | None = None,
+        proxy_user: str | None = None,
+        name: str = "arrow-spark",
+        num_executors: int | None = None,
+        status_poll_interval: int = 1,
+        env_vars: dict[str, Any] | None = None,
+        verbose: bool = False,
+        spark_binary: str | None = None,
+        properties_file: str | None = None,
+        yarn_queue: str | None = None,
+        deploy_mode: str | None = None,
+        use_krb5ccache: bool = False,
```

Comment:
Retrieve all arguments of `SparkSubmitOperator` except for `application` and `application_args`.

https://github.com/apache/airflow/blob/e827bfbd2c79f3432411e14351c245a06410766a/airflow/providers/apache/spark/operators/spark_submit.py#L38-L80